### PR TITLE
Frontend Improvements

### PR DESCRIPTION
**Pull request by winal-28**
Worked on some stuff on the frontend
image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -6,15 +6,18 @@
...
 <body>
+  <a href="https://github.com/wareneutron/psgc-api" target="_blank" id="fork-corner"
+    class="fork-corner fc-pos-tr fc-animate-grow fc-theme-github"></a>
   <main>
     <div class="main-content">
       <h1 class='logo'><span id="psgc">psgc</span>.<span id="rest">rest</span></h1>
       <div class="content">
         <h4><a href="https://docs.wareneutron.com/psgc-api"><span>Read the Official Documentation</span></a></h4>
-        <h4><a href="https://github.com/warengonzaga/fork-corner"class='github'>View Github Repository</a></h4>
         <h4>By <a href="#">Wareneutron Developers</a></h4>
```
Comment (WarenGonzaga) [inline public/index.html:21]: You forgot to add this link "https://github.com/wareneutron" as href value.

**Review by WarenGonzaga — CHANGES_REQUESTED** [review]
All good except the line I specify to update. Add the GitHub org link
https://github.com/wareneutron

File: public/index.html (hunk 1 above)
Comment (winal-28) [reply public/index.html:21]: ok

**Review by WarenGonzaga — APPROVED** [review]
All good! Ready to merge to the development branch. Thanks for the contribution!

**Pull request merged**